src/remote.js

```diff
@@ -259,20 +278,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
+              }
+            ),
+            function(error) {
+              util.err('Unable to create monaca project: ' + JSON.stringify(error));
             }
-          },
-          function(err) {
-            util.err("Unable to build this project " + err);
           }
-        )
-      },
-      function(err) {
-        util.err(err);
+        ),
+        function(err) {
+          util.err(err);
```

Comment:
@masahirotanaka Sorry, I forgot to report that it's related with this.

https://github.com/monaca/monaca-lib/pull/42#discussion_r51374845

You suggested that there was no need to stringify `err` at the origin, but it's actually needed in this case. Do you agree?